modules/ROOT/pages/browser-support.adoc

```diff
@@ -14,6 +14,13 @@ Use Anypoint Platform with the latest version of any supported browser:
 
 The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
 
+== Exceptions to Browser Support Policy
+
+API Community Manager and Partner Manager 2.0 each modify the browser support policy:
+
+* xref:api-community-manager::data-sheets.adoc[API Community Manager]
+* xref:release-notes::partner-manager/anypoint-partner-manager-2.0-release-notes#browser-compatibility.adoc[Partner Manager 2.0]
```

Comment:
I think this looks fine, but if you want to be really explicit, you could use something like:

The following Anypoint Platform products use modified versions of browser support:
* API Community Manager +
See xref:api-community-manager::data-sheets.adoc[API Community Manager].
* Partner Manager 2.0 +
See xref:release-notes::partner-manager/anypoint-partner-manager-2.0-release-notes#browser-compatibility.adoc[Partner Manager 2.0].